QMChatService/QMChatService/QMChatService.m

```diff
@@ -1250,7 +1250,7 @@ - (void)sendMessage:(QBChatMessage *)message
       saveToStorage:(BOOL)saveToStorage
          completion:(QBChatCompletionBlock)completion
 {
-    message.dateSent = [NSDate date];
+     if (!message.dateSent) message.dateSent = [NSDate date];
```

Comment:
@Pro100Andrey But why you are doing this? What if i want to use another time source, for example our server time?